src/applications/personalization/dashboard-2/components/claims-and-appeals/Claim.jsx

```diff
@@ -0,0 +1,73 @@
+import React from 'react';
+import moment from 'moment';
+
+import recordEvent from '~/platform/monitoring/record-event';
+import {
+  getPhaseDescription,
+  isClaimComplete,
+  getClaimType,
+} from '~/applications/claims-status/utils/helpers';
+
+const capitalizeFirstLetter = input => {
+  const capitalizedFirstLetter = input[0].toUpperCase();
+  return `${capitalizedFirstLetter}${input.slice(1)}`;
+};
+
+function listPhase(phase) {
+  return phase === 8 ? 'Closed' : getPhaseDescription(phase);
+}
+
+function handleViewClaim() {
+  recordEvent({
+    event: 'dashboard-navigation',
+    'dashboard-action': 'view-button',
+    'dashboard-product': 'view-claim',
+  });
+}
+
+const Claim = ({ claim }) => {
+  const inProgress = !isClaimComplete(claim);
+  const dateRecd = moment(claim.attributes.dateFiled).format('MMMM D, YYYY');
+  return (
+    <div className="vads-l-col--12 medium-screen:vads-l-col--6 small-desktop-screen:vads-l-col--8 medium-screen:vads-u-padding-right--3">
+      <div className="vads-u-padding-y--2p5 vads-u-padding-x--2p5 vads-u-background-color--gray-lightest">
+        <h3 className="vads-u-margin-top--0">
+          {capitalizeFirstLetter(getClaimType(claim))} claim received {dateRecd}
+        </h3>
+        <div className="vads-u-display--flex">
+          <i
+            aria-hidden="true"
+            className={`fas fa-fw fa-check-circle vads-u-margin-right--1 vads-u-margin-top--0p5 vads-u-color--green`}
+          />
+          <div>
+            <p className="vads-u-margin-y--0">
+              {listPhase(claim.attributes.phase)}
+            </p>
+            {inProgress && claim.attributes.developmentLetterSent ? (
```

Comment:
Just personal preference, instead of using a ternary, I would use `&&` to not have to return `null` -- or determine the copy as a variable at the top of the component to not have to repeat the `p` tag. But not too opinionated 😃

Reply:
- I get not liking the look of ternaries, but it's actually after to user them https://kentcdodds.com/blog/use-ternaries-rather-than-and-and-in-jsx
- it's possible to render multiple `p` tags depending on the claim data